### PR TITLE
feat: [CDS-73421]: Added triggerTypes and triggerIdentifiers in PipelineExecution filter

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
### Summary
This PR adds `triggerTypes` and `triggerIdentifiers`  fields for the PipelineExecution filter

#### Screenshots
Screenshot 2023-09-13 at 3 38 07 PM
Screenshot 2023-09-13 at 3 38 17 PM
Screenshot 2023-09-13 at 3 38 29 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

#### [Contributor license agreement](https://github.com/harness/harness-core-ui/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

[CDS-73421]: https://harness.atlassian.net/browse/CDS-73421?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by litvinovstas — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/70-pipeline/pages/execution-list/ExecutionListFilterForm/ExecutionListFilterForm.tsx [hunk 1]
```diff
@@ -40,14 +51,6 @@ interface ExecutionListFilterFormProps<T> {
...
 const NO_SELECTION = { label: '', value: '' }
 
-/*
-Examples - 
-  :v  ->  key: "", value: "v"
-  v:  ->  key: "v", value: ""
-  a:b ->  key: "a", value: "b"
-  a   ->  key: "a", value: null
-*/
-
```
Comment (rameshx) [inline src/modules/70-pipeline/pages/execution-list/ExecutionListFilterForm/ExecutionListFilterForm.tsx:50]: Why is this removed?

**Review by rameshx — APPROVED** [review]
